docs/products-solutions.asciidoc

```diff
@@ -5,8 +5,8 @@ The following Elastic products support ECS out of the box, as of version 7.0:
 
 * {beats-ref}/beats-reference.html[{beats}]
 * {apm-get-started-ref}/overview.html[APM]
-* {siem-guide}/siem-overview.html[Elastic SIEM app]
-** {siem-guide}/siem-field-reference.html[SIEM Field Reference Guide] - a list of ECS fields used in the SIEM app
+* {security-guide}/siem-overview.html[Elastic SIEM app]
```

Comment:
I've noticed that the new overview html page for the section is no longer `siem-overview.html` but is now called `es-overview.html`.

@lcawl I'll take over the adjustments to this PR and all backports, if you don't mind.

@benskelker With regards to file renames, do you know if `siem-field-reference.html` will be renamed as well? As far as I can tell that's still the name of the file. I'd like to know now if the name's going to change, in order to update all of these PRs only once 😅

Reply:
@webmat - yep that one will stay the same.